website/src/pages/Blog/index.tsx

```diff
@@ -161,16 +161,24 @@ const Blog: React.FC = () => {
     );
   };
 
+  const handleTabClickOnMobile = (newValue: string) => {
+    setValue(newValue);
+    setPage(1);
+  }
+
   const getTagsMarkup = Object.keys(tagsDistribution).map((item: string) => {
     if (tagsDistribution[item as keyof typeof tagsDistribution] > 1) {
       return (
-        <StyledTab
-          label={`${item}(${
-            tagsDistribution[item as keyof typeof tagsDistribution]
-          })`}
-          value={item}
-          key={item}
-        />
+        mediumViewport ? 
+          <StyledTab
+            label={`${item} (${tagsDistribution[item as keyof typeof tagsDistribution]})`}
+            value={item}
+            key = {item}
+          />
+          :
+          <Grid item xs={6} key={item}>
```

Comment:
IMO, one line comment would be good to have here, rest changes looks good  to me.

Reply:
Done